kmod/src/ioctl.c

```diff
@@ -516,7 +518,9 @@ static long scoutfs_ioc_stage(struct file *file, unsigned long arg)
 	}
 
 	si->staging = true;
+#ifdef KC_LINUX_BACKING_DEV_INFO
 	current->backing_dev_info = mapping->backing_dev_info;
+#endif
```

Comment:
I don't think this is right.  They removed the bdi from the address space, sure, but we're still required to set the task's bdi when it's calling high level vfs block functions.  Change the = mapping->.. to = inode_to_bdi(inode) and then leave the = NULL.   Take a peek at how/when other file systems also do this upstream.

Reply:
Yeah! got it.